cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -154,6 +154,69 @@ def __repr__(self) -> str:
 )
 
 
+def random_rotations_between_two_qubit_circuit(
```

Comment:
Is all below changes from the previous PR?

Reply:
Now that the previous is merged (and I merged master into this) these diffs should be all gone!